Bash_scripts/HW23/README.md

```diff
@@ -0,0 +1,36 @@
+# Ansible. Do for ubuntu and centos hosts.
+
+###  1.	Difference between ansible module: command, shell and raw
```

Comment:
What is the difference?

Reply:
Command works with python, if python is not installed on the host, then the command will not work.
Raw is a module that works at a low level of Linux, without using python scripts, which allows pre-configuration on hosts that do not have python installed.
Shell is the most convenient of all, it allows you to work with pipline, operands